cmd/controller-manager/app/controllermanager.go

```diff
@@ -146,7 +146,10 @@ func setupControllers(mgr controllerruntime.Manager, opts *options.Options, stop
 		Client:                       mgr.GetClient(),
 		EventRecorder:                mgr.GetEventRecorderFor(status.ControllerName),
 		PredicateFunc:                clusterPredicateFunc,
+		InformerManager:              informermanager.NewMultiClusterInformerManager(),
+		StopChan:                     stopChan,
 		ClusterClientSetFunc:         util.NewClusterClientSet,
+		ClusterDynamicClientSetFunc:  util.NewClusterDynamicClientSetForAgent,
```

Comment:
It seems like we should use NewClusterDynamicClientSet here, right?

Reply:
Done, thanks for reviewing.